Rivet/RivetSamples/Complete-Migration.ps1

```diff
@@ -0,0 +1,21 @@
+
+function Complete-Migration
+{
+    [CmdletBinding()]
+    [Rivet.Plugin([Rivet.Events]::AfterMigrationLoad)]
+    param(
+        [Parameter(Mandatory)]
+        # The migration the operation is part of.
+        [Rivet.Migration] $Migration
+    )
+
+    Set-StrictMode -Version 'Latest'
+    Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
+    
+    $problems = $false 
+
+    if( $problems )
+    {
+        throw ('There were errors running ''{0}''.  Please see previous errors for details.' -f $Migration.Name)

```

Comment:
Nit: Our [current PowerShell style](https://github.com/webmd-health-services/Standards/tree/main/PowerShell) is to do string interpolation like this:

```powershell
throw "There were errors running ""$($Migration.Name)"". Please see previous errors for details."
```